src/rpm.rs

```diff
@@ -382,7 +426,7 @@ where
 }
 
 impl Header<IndexSignatureTag> {
-    fn new_signature_header(size: i32, md5: &[u8], sha1: String) -> Self {
+    fn new_signature_header(size: i32, sha256checksum: &[u8], sha1: String) -> Self {
```

Comment:
I am not sure if this is too aggressive since the rpm file format specifically states md5. I think it would be best to have multiple constructors since we may encounter legacy packages with strict md5 requirements.

Reply:
After reading a couple of sources again (to be precise, the impl used by rpm itself) I think this _has_ to be md5 to even work.